src/app/ApplicationTemplate.Access/ApiClients/Authentication/AuthenticationApiClientMock.cs

```diff
@@ -45,30 +45,26 @@ public async Task<AuthenticationData> RefreshToken(CancellationToken ct, Authent
 
 		await SimulateDelay(ct);
 
-		return CreateAuthenticationData(unauthorizedToken.AccessTokenPayload);
+		return CreateAuthenticationData(unauthorizedToken.AccessToken.Payload);
 	}
 
-	private AuthenticationData CreateAuthenticationData(AuthenticationToken token = null, TimeSpan? timeToLive = null)
+	private AuthenticationData CreateAuthenticationData(AuthenticationToken token = null)
 	{
-		var encodedJwt = CreateJsonWebToken(token, timeToLive);
-		var jwt = new JwtData<AuthenticationToken>(encodedJwt, _serializerOptions);
+		var encodedJwt = CreateJsonWebToken(token);
 
 		return new AuthenticationData()
 		{
-			AccessToken = jwt.Token,
+			AccessToken = new JwtData<AuthenticationToken>(encodedJwt, _serializerOptions),
```

Comment:
Is there a reason we prefer doing it this way?

Reply:
Oh, basically, I just fall back to what was previously used. I'll have to confirm with @jeanplevesque, I think he once told me we were not using it anymore due to an issue with .NET 6 that has been solved since.